src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,117 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int OFFSET = 1;
+    private static final int SIZE_ADJUSTMENT = 1;
+    private static final int START_INDEX = 0;
+    private static final double ELEMENT_DATA_MULTIPLIER = 1.5;
+    private int listCapacity = 10;
```

Comment:
Any magic numbers should be constants. The initial capacity of the list (10) should be extracted to a constant with an informative name, such as INITIAL_CAPACITY.